content/blog/2018-09-04-birds-taxo-traits.md

```diff
@@ -0,0 +1,541 @@
+  dplyr::select(id, threat2, threat1) %>%
+  unique() %>%
+  dplyr::count(threat2, threat1) %>%
+  dplyr::arrange(-n) %>%
+  head(n = 5) %>%
+  knitr::kable()
+```
+
+| threat2                                | threat1                         |    n|
+|:---------------------------------------|:--------------------------------|----:|
+| Hunting & trapping terrestrial animals | Biological resource use         |   25|
+| Habitat shifting & alteration          | Climate change & severe weather |   24|
+| Renewable energy                       | Energy production & mining      |   16|
+| Agricultural & forestry effluents      | Pollution                       |   14|
+| Dams & water management/use            | Natural system modifications    |   14|
+
+It’s quite similar the ranking in this post on [BirdLife’s
+website](https://www.birdlife.org/worldwide/news/top-five-threats-birds-may-surprise-you):
+industrial farming, logging, invasive species, hunting and trapping,
+climate change. Note that these threats have been summarized by species,
+not by species and location, so this data doesn’t tell us much about the
+state of each species in the County of Constance. If we wanted to
+characterize the County a bit more via the use of open data, we could
+use e.g. `osmdata` to get land-use information via OpenStreetMap:
+instead of bird hides/blinds as in the first post we could get elements
+related to agriculture for instance.
+
+### Conclusion
+
+#### Characterizing the local birds population
+
+From occurrence data we got names of species observed at least once in
+the area over the last year. Using taxonomy data to classify them we
+were able to see quite a few birds orders are represented, and we were
+able to visualize corresponding silhouettes. Traits data is general and
+doesn’t give information about the local context of birds in the County
+of Constance, however it could be coupled with more local data:
+localization of observations, and open geographical data. All the
+mentioned data sources are available for free, and can be obtained using
+R packages.
+
+#### Phylogeny and traits data in R
+
+There are many R packages supporting your phylogeny work! In particular,
+within the rOpenSci suite `taxize` that we used here allows to get
+taxonomy information from many sources, while
+[`taxa`](https://github.com/ropensci/taxa) defined taxonomy classes for
+R. The [`treeio` package](https://github.com/GuangchuangYu/treeio) by
+Guangchuang Yu provides base classes and functions for phylogenetic tree
+input and output and was onboarded. Explore more of our packages suite,
+including and beyond the geospatial category,
+[here](https://ropensci.org/packages/).
+
+Mention `taxize` book?
+
+Guangchuang Yu’s other packages such as `ggtree` and `tidytree` are also
+of interest for manipulating and visualizing trees. More generally, the
+[Phylogenetics CRAN task
+view](https://cran.r-project.org/web/views/Phylogenetics.html) provides
```

Comment:
there's a taxonomy task view as well but it's not on cran https://github.com/ropensci/taxonomy